students/swietczak_monika/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,61 @@
+def calculate(option, a, b):
+    """
+    Performs calculations according to selected option
+    :param option: string, one of selected options
+    :param a: integer
+    :param b: integer
+    :return: number
+    """
+    if option == "a":
+        return a + b
+    if option == "s":
+        return a - b
+    if option == "m":
+        return a * b
+    if option == "d":
+        return a / b
+    if option == "p":
+        return a ** b
+
+
+def calculator(option):
+    """
+    Main function displaying info/result/help to user. It also runs calculate
+    function if user selects the right option.
+    :param option: String - selected option, one of values: a,s,m,d,p,h,?,q
+    """
+    try:
+        if option == "a":
+            print("ADDING")
+        if option == "s":
```

Comment:
For code readability it's better to use elif here and handle default case with else when incorrect option was provided